src/Locale.php

```diff
@@ -132,23 +139,28 @@ private function getLocaleFromPath(string $path): array
 
         $pattern = implode('|', $parts);
         if (preg_match("#^/($pattern)\b(/?)#i", $path, $matches)) {
-            $locale = $matches[1];
-            [$locale, $country] = $this->parseLocale($locale);
-            if (isset($this->locales[$locale])) {
+            $matchedLocale = $matches[1];
+            [$locale, $country] = $this->parseLocale($matchedLocale);
+            if (isset($this->locales[$locale]) || in_array($matchedLocale, $this->locales, true)) {
```

Comment:
Do we store `$this->locales` as

```php
[
  'en',
  'kz' => ...
]
```

?

i.e. both locales as keys and locales as values?

Reply:
Is it possible to always have locales as keys? This way lookup will be O(1) instead of O(N).